docs/web/docs/1-getting_started/3-aws.mdx

```diff
@@ -197,35 +198,35 @@ properties:
         principal:
           service: ec2.amazonaws.com
   inline_policies:
-    - policy_name: spoke-acct-policy
+    - policy_name: s3-access
       statement:
-        - # excluded_accounts: # Include the policy on the role across all accounts, except ACCOUNT_A
-          #  - ACCOUNT_A
-          action:
-            - s3:ListBucket
-          effect: Deny
-          resource: "*"
-        - # included_accounts: # Only include the policy statement on ACCOUNT_A
-          # - ACCOUNT_A
+        - # excluded_accounts: # Include the policy on the role across all accounts, except PROD_ACCOUNT
+          #  - PROD_ACCOUNT
           action:
             - s3:GetObject
-          effect: Deny
-          resource: "*"
+          effect: Allow
+          resource: "arn:aws:s3:::awsexamplebucket1-{{var.account_name}}/*"
+        - # included_accounts: # Only include the policy statement on DEV_ACCOUNT
+          # - DEV_ACCOUNT
+          action:
+            - s3:DeleteObject
+          effect: Allow
+          resource: "arn:aws:s3:::awsexamplebucket1-{{var.account_name}}/*"
         - included_accounts:
             - "*"
-          # Include the policy statement on all accounts except ACCOUNT_A and ACCOUNT_B
+          # Include the policy statement on all accounts except AUDITOR_ACCOUNT and SECURITY_ACCOUNT
           # excluded_accounts:
-          #   - ACCOUNT_A
-          #   - ACCOUNT_B
+          #   - AUDITOR_ACCOUNT
+          #   - SECURITY_ACCOUNT
           action:
             - s3:ListAllMyBuckets
-          effect: Deny
+          effect: Allow
           resource: "*"
   managed_policies:
-    - policy_arn: arn:aws:iam::aws:policy/job-function/ViewOnlyAccess
+    - policy_arn: arn:aws:iam::aws:policy/job-function/SupportUser
   path: /iambic_test/
   permissions_boundary:
-    policy_arn: arn:aws:iam::aws:policy/AWSDirectConnectReadOnlyAccess
+    policy_arn: arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess
```

Comment:
As a global permission boundary, AmazonS3ReadOnlyAccess would negate the permissions we're trying to grant on lines 212 because the maximum permission that any role can have is the s3 read permissions

Reply:
Cool, gonna swap it with AmazonS3FullAccess